examples/kafka_prometheus/main.tf

```diff
@@ -0,0 +1,59 @@
+terraform {
+  required_providers {
+    aiven = "2.0.1"
+  }
+}
+
+provider "aiven" {
+  api_token = var.aiven_api_token
+}
+
+data "aiven_project" "bd" {
+  project = var.project
+}
+
+
+resource "aiven_service" "bar" {
```

Comment:
Please run `terraform fmt` to format HCL code, it will fix different indentation issue.